eventmesh-runtime/src/main/java/org/apache/eventmesh/runtime/core/protocol/tcp/client/task/GoodbyeTask.java

```diff
@@ -58,13 +58,10 @@ public void run() {
             msg.setHeader(new Header(CLIENT_GOODBYE_RESPONSE, OPStatus.FAIL.getCode(), Arrays.toString(e.getStackTrace()),
                 pkg.getHeader().getSeq()));
         } finally {
-            this.eventMeshTCPServer.getScheduler().submit(new Runnable() {
+            this.eventMeshTCPServer.getScheduler().submit(() -> {
```

Comment:
It should be modified like this：
this.eventMeshTCPServer.getScheduler().submit(() -> Utils.writeAndFlush(msg, startTime, taskExecuteTime, session.getContext(), session));